### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by juanespinola05**
## What?
Added:
- Footer component
- Hyperlink component with _blank target 
- Facebook, email and instagram icons with handler
- Add Poppins and Mulish fonts from google fonts
- Logo component
## Anything Else?
Navbar is temporal because its component hasn't been created yet
It's not positioned at the bottom of the page
Social Media links and logo image should be getted from an endpoint (waiting for redux saga setup)

### CODE REVIEW

File: src/components/Logo/index.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+function Logo({ size = '100', ...props }) {
+  const styles = {
+    width: `${size}px`,
+  };
+
+  return (
+    <img
+      style={styles}
+      src={logo}
+      alt='Somos Más Logo'
```
Comment (nicolasgonzalezgonzanlez) [inline src/components/Logo/index.jsx]: El alt que se aparte de las props

File: src/components/Icon/index.jsx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export { default as FacebookIcon } from './FacebookSVG';
```
Comment (nicolasgonzalezgonzanlez) [inline src/components/Icon/index.jsx]: Todas las imagenes llevarla a una carpeta assets

File: src/components/BlankAnchor/index.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const AnchorBlank = ({ href = '/', children, ...props }) => (
+  <a
```
Comment (nicolasgonzalezgonzanlez) [inline src/components/BlankAnchor/index.jsx:4]: Tiene que ser el component LInk de react router

Comment (nicolasgonzalezgonzanlez) [reply src/components/BlankAnchor/index.jsx:4]: Renombrar el componente

File: src/components/Footer/index.jsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+function Footer() {
+  return (
+    <footer className={`${styles.footer} text-center position-relative pt-5`}>
+      <div className={`${styles.footer__logo} position-absolute start-50 translate-middle`}>
+        <Logo size='140' />
+      </div>
+      <div className={`${styles.footer_navbar} pt-5`}>
+        <nav className='container'>
+          <ul>
+            <li className='d-inline-block mx-3'>
```
Comment (nicolasgonzalezgonzanlez) [inline src/components/Footer/index.jsx:22]: Podes crear un component Li que sea parte de los componentes del footer y los parametros manejarlo desde un array

File: src/components/Icon/FacebookSVG.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const FacebookIcon = ({ size }) => (
```
Comment (nicolasgonzalezgonzanlez) [inline src/components/Icon/FacebookSVG.jsx]: Para estos iconos, crear un componente generico que revisa el svg y no crear componentes para cada svg

File: src/components/BlankAnchor/index.jsx (hunk 3 above)
Comment (juanespinola05) [reply src/components/BlankAnchor/index.jsx:4]: Son links externos al sitio

File: src/components/Logo/index.jsx (hunk 1 above)
Comment (juanespinola05) [reply src/components/Logo/index.jsx]: Hecho

File: src/components/Icon/index.jsx (hunk 2 above)
Comment (juanespinola05) [reply src/components/Icon/index.jsx]: Hecho

File: src/components/Icon/FacebookSVG.jsx (hunk 5 above)
Comment (juanespinola05) [reply src/components/Icon/FacebookSVG.jsx]: Hecho

File: src/components/Footer/index.jsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (nicolasgonzalezgonzanlez) [inline src/components/Footer/index.jsx:3]: No va aqui, va en la raiz del proyecto

**Comment by juanespinola05** [comment]
Now that redux saga is setup, I created a saga for the ong public data so the footer and logo components consume it from an API. This api endpoint was created with mockapi.io as the backend endpoint does not exist yet

**Review by nicolasgonzalezgonzanlez — APPROVED** [review]

**Pull request merged**